src/SharedPipelineComponents/Datalock/SFA.DAS.CollectionEarnings.DataLock/Domain/Extensions/CommitmentExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace SFA.DAS.CollectionEarnings.DataLock.Domain.Extensions
+{
+    public static class CommitmentExtensions
+    {
+        public static bool WithdrawnBeforeStart(this Commitment source)
+        {
+            if (source.WithdrawnOnDate < source.EffectiveFrom)
```

Comment:
does time matter here? 
```suggestion
            if (source.WithdrawnOnDate.Date < source.EffectiveFrom.Date)
```